swatch-metrics/build.gradle

```diff
@@ -8,8 +8,8 @@ dependencies {
     implementation 'io.quarkus:quarkus-config-yaml'
     implementation 'io.quarkus:quarkus-logging-json'
     implementation 'io.quarkus:quarkus-opentelemetry'
-    implementation 'io.quarkus:quarkus-resteasy-reactive-jsonb'
-    implementation 'io.quarkus:quarkus-rest-client-reactive-jsonb'
+    implementation 'io.quarkus:quarkus-resteasy-reactive-jackson'
+    implementation 'io.quarkus:quarkus-rest-client-reactive-jackson'
```

Comment:
I had to use `jackson` because the events model is generated using jackson and I had issues with enum serialization (the generated json was incompatible with the one required by the swatch tally service).  

I think we should always use either jackson (this is my option) or jsonb (which is used by the rest-client for Quarkus), but nowadays we use both.

Reply:
Reported in https://issues.redhat.com/browse/SWATCH-1945